docs/usage/cli/filter.rst

```diff
@@ -63,3 +71,184 @@ The filter command allows you to select a specific number of sequences from spec
+first two steps, read through it once in the final step.
+
+Stitching calls together with Snakemake
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The approach above can be cumbersome with more intermediate samples. There is a
+more flexible approach using `Snakemake`_.
+
+1. Add a section in the `config file`_.
+
+  .. code-block:: yaml
+
+    subsampling:
+      north_america: --query "region == 'North America'" --subsample-max-sequences 20
+      europe: --query "region == 'Europe'" --subsample-max-sequences 50
+
+2. Add two rules in a `Snakefile`_.
+
+  .. code-block:: python
+
+    # 1. Sample 20 sequences from North America
+    # 2. Sample 50 sequences from Europe
+    rule intermediate_sample:
+        input:
+            metadata = "data/metadata.tsv",
+        output:
+            strains = "results/{sample_name}_sample_strains.txt",
+        params:
+            augur_filter_args = lambda wildcards: config.get("subsampling", {}).get(wildcards.sample_name, "")
+        shell:
+            """
+            augur filter \
+                --metadata {input.metadata} \
+                {params.augur_filter_args} \
+                --output-strains {output.strains}
+            """
+
+    # 3. Combine using augur filter
+    rule combine_intermediate_samples:
+        input:
+            sequences = "data/sequences.fasta",
+            metadata = "data/metadata.tsv",
+            intermediate_sample_strains = expand("results/{sample_name}_sample_strains.txt", sample_name=list(config.get("subsampling", {}).keys()))
+        output:
+            sequences = "results/combined.fasta",
+            metadata = "results/combined.tsv",
+        shell:
+            """
+            augur filter \
+                --sequences {input.sequences} \
+                --metadata {input.metadata} \
+                --exclude-all \
+                --include {input.intermediate_sample_strains} \
+                --output-sequences {output.sequences} \
+                --output-metadata {output.metadata}
+            """
+
+3. Run Snakemake targeting the second rule.
+
+  .. code-block:: bash
+
+    snakemake --forcerun combine_intermediate_samples
+
+Explanation:
+
+- The configuration section consists of one entry per intermediate sample in the
+  format ``sample_name: <augur filter arguments>``.
+- The first rule is run once per intermediate sample using `wildcards`_ and an
+  `input function`_. The output of each run is the sampled strain list.
+- The second rule uses `expand()`_ to define input as all the intermediate
+  sampled strain lists, which are passed directly to ``--include`` as done in
+  the previous example.
+
+It is easy to add or remove intermediate samples. Example:
+
+.. code-block:: diff
+
+  subsampling:
+  -  north_america: --query "region == 'North America'" --subsample-max-sequences 20
+     europe: --query "region == 'Europe'" --subsample-max-sequences 50
+  +  asia: --query "region == 'Asia'" --subsample-max-sequences 30
```

Comment:
These examples of how easy it is to add/remove samples are a great idea. Since readers who aren’t familiar with diff syntax may mistake the contents of this code block as something to be copied directly into their build configs, what if we dropped the `diff` syntax view in favor of two separate specific alternate files?

Reply:
Done: 05773c5b92d084ce75ad2fc2f9873b3bafe47837